instantsearch-smart-categories-widget/README.md

```diff
@@ -0,0 +1,102 @@
+## Smart category widgets
+
+* [Overview](#overview)
```

Comment:
The structure doesn't follow the [README template](https://github.com/algolia/instantsearch-labs/blob/283baddeb9ae4a17d2f30452f310b8eb7bf109f5/example/instantsearch.js/README.md).